database/user_models.py

```diff
@@ -185,7 +185,7 @@ def create_without_password(cls, username):
         Create a new Researcher with provided username and no password
         """
 
-        r = cls(username=username, password='fakepassword', salt='cab', admin=False)
+        r = cls(username=username, password='fakepassword', salt='cab', site_admin=False)
```

Comment:
Looks like 'admin' was renamed to 'site_admin'; perhaps in [this commit](https://github.com/danielnorton/beiwe-backend/commit/cda3d19ec08025854d6843a2b7cc3d1e83470027).